src/System.Private.CoreLib/shared/System/Guid.cs

```diff
@@ -127,49 +124,35 @@ private enum GuidParseThrowStyle : byte
             AllButOverflow = 2
         }
 
-        // This will store the result of the parsing.  And it will eventually be used to construct a Guid instance.
+        // This will store the result of the parsing. And it will eventually be used to construct a Guid instance.
         private struct GuidResult
         {
-            internal readonly GuidParseThrowStyle _throwStyle;
+            private readonly GuidParseThrowStyle _throwStyle;
             internal Guid _parsedGuid;
 
-            private bool _overflow;
-            private string _failureMessageID;
-            private object _failureMessageFormatArgument;
-
             internal GuidResult(GuidParseThrowStyle canThrow) : this()
             {
                 _throwStyle = canThrow;
             }
 
-            internal void SetFailure(bool overflow, string failureMessageID)
+            internal void SetFailure(bool overflow, string failureMessageID, string failureMessageFormatArgument = null)
```

Comment:
Could you please to either delete the `failureMessageFormatArgument` argument here and in all places that pass it in; or fix the code to actually use it.